crates/astria-sequencer/src/app.rs

```diff
@@ -139,7 +139,12 @@ impl App {
             .await
             // TODO: explain why this is an invariant of the system
             .expect("block height should be set");
-        info!(?tx, ?height, "executed transaction");
+        info!(
+            ?tx,
```

Comment:
It's ok to leave this with `?`. But can you make a follow up issue that we replace this with a json wrapper?